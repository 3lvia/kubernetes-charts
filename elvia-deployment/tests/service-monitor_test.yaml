suite: test service monitor
templates:
  - service-monitor.yaml
tests:
  - it: service monitor snapshot should match
    set:
      image.tag: latest
      name: demo-api
      namespace: core
      microserviceType: webapi
      environment: dev
      replicaCount: 1
      resources:
        requests:
          cpu: 100m
          memory: 128Mi
        limits:
          cpu: 400m
          memory: 500Mi
    asserts:
      - matchSnapshot: {}
