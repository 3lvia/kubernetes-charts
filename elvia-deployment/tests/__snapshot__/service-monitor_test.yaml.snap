service monitor snapshot should match:
  1: |
    apiVersion: monitoring.coreos.com/v1
    kind: ServiceMonitor
    metadata:
      name: demo-api
      namespace: core
    spec:
      endpoints:
        - port: demo-apiport
      selector:
        matchLabels:
          app: demo-api
